.goreleaser.yaml

```diff
@@ -207,7 +207,14 @@ snapcrafts:
       safely and efficiently. OpenTofu can manage existing and popular service
       providers as well as custom in-house solutions.
     base: core22
-    grade: "{{ .Env.SNAPCRAFT_CHANNEL }}"
+    grade: '{{ if eq .Env.RELEASE_FLAG_PRERELEASE "true" }}devel{{ else }}stable{{ end }}'
+    channel_templates:
+      - '{{ .Major }}.{{ .Minor }}/{{ if eq .Env.RELEASE_FLAG_PRERELEASE "true" }}edge{{ else }}stable{{ end }}'
```

Comment:
Right now we only have the latest track, so I believe we should only be referencing stable / edge here, and that's it (which will translate to latest/stable and latest/edge).

It seems like requesting new tracks (like 1.6) is a moderately involved process, and requires a forum thread on the snapcraft forum for each.

Reply:
@bunniseng

Reply:
Okay, can you confirm with me the requirements as I understand it in your comments.

We want to release to the `latest/edge` channel when the "Release as prerelease" option has been selected otherwise release to the `latest/stable` channel.

This also means we will no longer have the need for the additional input "Release as latest" for now and it can be removed.

@cube2222

Reply:
I think in both cases we also need "Release as latest" to be true, so I'd keep it.

E.g. if someone releases 1.6.3 when 1.7.0 is out already, it's not latest, so it doesn't go to snapcraft at all.

Reply:
Okay I think I understand, can you look over the scenarios below and confirm.

Given: "release as prerelease" and "release as latest" is selected
When: the release pipeline runs
Then: the snap is published to the `latest/edge` channel

Given: "release as latest" is selected
When: the release pipeline runs
Then: the snap is published to the `latest/stable` channel

Given: "release as latest" is not selected
When the release pipeline runs
Then: the snap is not published